Tests/RichTextKitTests/RichTextViewRepresentableTests.swift

```diff
@@ -45,9 +45,10 @@ final class RichTextViewComponentTests: XCTestCase {
     func testSettingUpWithEmptyTextWorks() {
         let string = NSAttributedString(string: "")
         view.setup(with: string, format: .rtf)
+        view.configuration = .standard
```

Comment:
This raises question if configuration shouldn't be passed to setup instead... This feels like API is all over the place, your thoughts @danielsaidi ?

Reply:
I'm thinking we should inject it into the environment in SwiftUI.